### PR TITLE
Update messages.php

### PR DESCRIPTION
**Pull request by sunithamundrathi**
Hey Xingyi 

can you review the code and let me know the necessary changes?

### CODE REVIEW

File: messages.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?php
+// Connect to the database
+$servername = "localhost";
+$username = "dev640";
+$password = "dev640password";
+$dbname = "dev640socialapp";
+$conn = new mysqli($servername, $username, $password, $dbname);
```
Comment (xingyicc) [inline messages.php:7]: No need to connect to the database here. We do database connection in header.php. You should require header.php in this page.

File: messages.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+$servername = "localhost";
+$username = "dev640";
+$password = "dev640password";
+$dbname = "dev640socialapp";
+$conn = new mysqli($servername, $username, $password, $dbname);
+
+// Check connection
+if ($conn->connect_error) {
+    die("Connection failed: " . $conn->connect_error);
+}
```
Comment (xingyicc) [inline messages.php]: No need to connect to the database here. We do database connection in header.php. You should require header.php in this page.

File: messages.php
Comment (xingyicc) [inline messages.php:1]: We expected at least 2 functions: add/view. What I'm getting is an empty page. After refreshing, it shows no message. 
You can check the example page: https://github.com/RobinNixon/lpmj6/blob/master/robinsnest/messages.php

**Review by xingyicc — CHANGES_REQUESTED** [review]

**Comment by sunithamundrathi** [comment]
hi Xingyi

Can you review the updated changes and let me know if anything is needed?

thanks 
Sunitha Mundrathi

**Pull request merged**